rollup.config.js

```diff
@@ -0,0 +1,29 @@
+import typescript from 'rollup-plugin-typescript2';
+import { terser } from "rollup-plugin-terser";
+import * as path from "path";
+
+const BASE_PATH = process.cwd();
+const getFile = path => path.resolve(BASE_PATH, path);
+
+const pkg = require(getFile('package.json'));
+
+export default {
+  input: path.resolve(BASE_PATH, 'src/index.ts'),
+  external: Object.keys({ ...pkg.dependencies, ...pkg.peerDependencies }),
```

Comment:
Hmm, this is true for libraries that are run in NodeJs — probably not good for frontend libraries (such as ui-kit). Is this on purpose? This should be specified on a project basis I think.

Reply:
Yeah, good point.

Reply:
Should I read `rollup.config.js` from current project then and merge the config?

Reply:
Yeah, I think that'd be the best solution.